src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,79 @@
-import React from 'react';
+import cn from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { RootState } from '../../app/store';
+import { actions } from '../../features/currentTodo';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loader, setLoader] = useState(true);
+  const selectTodo = useAppSelector((state: RootState) => state.currentTodo);
```

Comment:
```suggestion
  const selectedTodo = useAppSelector((state: RootState) => state.currentTodo);
```
selectTodo sounds like a function name

Reply:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/f42eba39-32d7-40e0-9391-08a107037076">